core/src/commonMain/kotlin/ua/syt0r/kanji/presentation/screen/main/screen/writing_practice/ui/WritingPracticeInfoSection.kt

```diff
@@ -276,6 +299,14 @@ private fun AnimatedCharacterSection(
 
 }
 
+@Composable
+private fun StrokeCountText(count: Int) {
+    Text(
+        text = resolveString { writingPractice.strokesMessage(count) },
+        style = MaterialTheme.typography.bodyMedium.plus(TextStyle(color = Color.Gray))
```

Comment:
Please set text color using color argument, not by modifying style. Also use color from theme like `MaterialTheme.colorScheme.onSurface` or something, you can use `copy(alpha =)` method of Color to make it less visible if available colors are not suitable